src/main/java/seedu/address/logic/commands/client/EditClientCommand.java

```diff
@@ -14,10 +25,55 @@ public class EditClientCommand extends ClientCommand {
 
     public static final String MESSAGE_SUCCESS = "Edited Client: %1$s";
```

Comment:
Would be good for the success message to follow the same format as project and issue

Reply:
Noted and changed. Thanks!